### PR TITLE
[FHTML-22] Revision log

### PR DESCRIPTION
**Pull request by blakey87**
New revision log component added, this will show Drupal revisions, the list of logs can be opened and closed, using the button on the top right.
image

## Checklist before opening a PR

Before opening a PR, please make sure:

- [x] your work is in a branch based on and synced with the develop branch.
- [x] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [x] your newComponent is imported in the index.js file.
- [x] your folder hierarchy follows the proper structure.
- [x] your code follows the coding standards and naming conventions defined.
- [x] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [x] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [x] your code has been tested in all the different screensizes defined.
- [x] you have run the `yarn lint` command and fixed any linting errors.
- [x] you have updated the CHANGELOG.md file with your changes.

### CODE REVIEW

File: src/components/article/RevisionLogBottom/revisionLogBottom.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <ul class="revision-log__link-list" aria-hidden="false">
+      {% for log in logs %}
+        <li class="revision-log__link-list__item" id="fn:{{ log.key }}">
+          <strong class="revision-log__link-list__item-date">{{ log.date }}</strong>
+          <span  class="revision-log__link-list__item-text">{{ log.text }}</span>
+        </li>
+      {% endfor %}
+    </ul>
+</div>
```
Comment (c-ancia) [inline src/components/article/RevisionLogBottom/revisionLogBottom.html.twig:23]: can you fix the indentation on this file please? Should be 2 spacings.

Comment (c-ancia) [inline src/components/article/RevisionLogBottom/revisionLogBottom.html.twig:23]: should we wrap this component in a {% if logs%} or is it handled on Drupal side? 🤔

**Review by c-ancia — CHANGES_REQUESTED** [review]

File: src/components/article/RevisionLogBottom/revisionLogBottom.scss [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    margin-top: 0.5rem;
+
+    p {
+      margin: 0;
+    }
+  }
+
+  &__toggle-button {
+    display: none;
+    background: #fff;
```
[the remark this answers is not in the document]
Comment (blakey87) [reply src/components/article/RevisionLogBottom/revisionLogBottom.scss]: Sure pushed the update.

File: src/components/article/RevisionLogBottom/revisionLogBottom.html.twig (hunk 1 above)
Comment (blakey87) [reply src/components/article/RevisionLogBottom/revisionLogBottom.html.twig:25]: How did I miss this!
Pushed the update.

File: CHANGELOG.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Add external link example to the subject listing section
 - Site map component
+- Add new revision log component
```
Comment (c-ancia) [inline CHANGELOG.md:52]: can you move it to the new "upcoming release" part please? :)

**Review by c-ancia — APPROVED** [review]

**Pull request merged**